.github/workflows/pkg-config.yaml

```diff
@@ -0,0 +1,51 @@
+name: pkg-config
+
+permissions:
+  contents: read
+
+on:
+  push:
+  pull_request:
+  merge_group:
+  schedule:
+    - cron: '15 12 * * 3'
+
+jobs:
+  build:
+    name: Build+test
+    runs-on: ubuntu-latest
```

Comment:
I think we can add macos to the matrix by:

1. using `brew` to install `pkg-config`
2. switching out the `CARGO_C_FILE` var in the "install cargo-c"  step w/ the macos binary release.

I ran out of time to implement that this evening but will consider it as follow-up.

Reply:
Filed https://github.com/rustls/rustls-ffi/issues/424 to not forget this.